lib/DBIx/Class/Helper/ResultSet/Shortcut/ResultsExist.pm

```diff
@@ -24,11 +24,14 @@ sub results_exist_as_query {
 sub results_exist {
    my $self = shift;
 
+   my $query = $self->results_exist_as_query;
+   $$query->[0] .= 'AS _existence_subq';
```

Comment:
Actually one last-last nit. I'd recommend doing `' AS ...` here, to leave an explicit space between the closing `)` and the `AS`. Generally it *shouldn't* matter, but who knows...

I.e. like in https://github.com/frioux/DBIx-Class-Helpers/issues/54#issuecomment-563376551

Reply:
k, that's coming up in the next commit